sale_order_line_packaging_qty/models/sale_order_line.py

```diff
@@ -35,16 +35,14 @@ def _compute_product_packaging_qty(self):
                 )
             else:
                 product_qty = sol.product_uom_qty
+            rounding = sol.product_packaging.product_uom_id.rounding or 2
```

Comment:
@acsonefho Why not using uom from the line instead and why 2?

https://github.com/odoo/odoo/blob/13.0/addons/sale/models/sale.py#L1284

```suggestion
            rounding = sol.product_packaging.product_uom_id.rounding or sol.product_uom or sol.product_id.uom_id
```